CONTRIBUTING.md

```diff
@@ -909,6 +910,34 @@ test cases.
 you should name it ``module_name_example`` (ie: `cms_form` and `cms_form_example`).
 In this way coverage analysis will ignore this extra module by default.
 
+### Investigating Travis Test Failures
+
+It can sometimes be difficult to reproduce a Travis test failure locally due to
+subtle environment differences. In these scenarios it can be helpful to connect to
+the Runbot container generated for that branch/PR via SSH, where the
+environment will be very similar to Travis. You can do this by running:
+
+```
+ssh -p [port] odoo@runbot[1 or 2].odoo-community.org
+```
+
+The correct Runbot subdomain can be found by checking the info on 
+https://runbot.odoo-community.org/runbot for your particular repo and branch.
+The port can also be found there by clicking on the gear icon next to the
+relevant Runbot instance and adding 1 to the port number in the dropdown. In
+order to be authenticated, your public SSH key will need to be associated with
+your GitHub account **before** the Runbot instance is generated.
+
```

Comment:
What about mentions the non stop the default odoo service restriction?
In order to avoid a container stopped.